liblangutil/Token.h

```diff
@@ -88,6 +88,7 @@ namespace langutil
 	/* Assignment operators. */										\
 	/* IsAssignmentOp() relies on this block of enum values being */	\
 	/* contiguous and sorted in the same order!*/						\
+	T(AssemblyAssign, ":=", 2)                                                   \
```

Comment:
If you put it here, it is not in line with the documentation above anymore. Better to put it somewhere else.